### PR TITLE
OSOE-616: Add UI tests in Lombiq.OrchardCoreApiClient

### PR DESCRIPTION
**Pull request by Piedone**
[OSOE-616](https://lombiq.atlassian.net/browse/OSOE-616)

[OSOE-616]: https://lombiq.atlassian.net/browse/OSOE-616?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj [hunk 1]
```diff
@@ -40,7 +40,6 @@
     <PackageReference Include="Lombiq.Hosting.Tenants.Management" Version="5.0.0" />
     <PackageReference Include="Lombiq.Hosting.Tenants.MediaStorageManagement" Version="5.0.0" />
     <PackageReference Include="Lombiq.JsonEditor" Version="5.0.0" />
-    <PackageReference Include="Lombiq.OrchardCoreApiClient" Version="3.0.0" />
```
Comment (wAsnk) [inline NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj:43]: Why was this removed?

Comment (Piedone) [reply NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj:43]: That project doesn't do anything in itself and makes no sense to have it referenced. It's only needed when the API Client is actually used, and that only happens in the UI test project. There it doesn't need to be referenced directly though, since it's a dependency of `Lombiq.OrchardCoreApiClient.Tests.UI`.

**Review by wAsnk — APPROVED** [review]
